navigation.js

```diff
@@ -0,0 +1,52 @@
+const links = document.querySelectorAll('.nav-link');
+
+links.forEach((link) => {
+  link.addEventListener('click', (e) => {
+    e.preventDefault();
+    const id = link.getAttribute('href').slice('1');
+
+    const currentSection = document.getElementById(id);
+    const blockElement = document.querySelector('.block');
+    blockElement.classList.remove('block');
+    blockElement.classList.add('none');
+    currentSection.classList.remove('none');
+    currentSection.classList.add('block');
+  });
+});
+
+function checkTime(i) {
+  if (i < 10) {
+    i = `0${i}`;
+  }
+  return i;
+}
+
+function startTime() {
+  const monthNames = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+  const today = new Date();
+  const k = today.getMonth();
+  const l = today.getDate();
+  const n = today.getFullYear();
+  const h = today.getHours();
+  let m = today.getMinutes();
+  let s = today.getSeconds();
+  m = checkTime(m);
+  s = checkTime(s);
+  document.getElementById('time').innerHTML = `${monthNames[k]} ${l} ${n}, ${h}:${m}:${s}`;
+  setTimeout(startTime, 1000);
+}
+
+startTime();
```

Comment:
- [ ] The current date is displayed below the navigation bar, for this requirement, students should use the  [Luxon](https://moment.github.io/luxon/api-docs/index.html) library. See Get the current date and time
=> This requirement is not fully implemented, the date and time is present but you have to use [Luxon](https://moment.github.io/luxon/api-docs/index.html) as it was required in project requirement, try that out so that it will help you learn more about different libararies